events/class_tests/create_events.py

```diff
@@ -42,7 +42,9 @@ class TestCreateEventForm:
             {'title': TITLE, 'date_time_start': DATE_TIME_END, 'date_time_end': DATE_TIME_START, 'color': COLOR},
             # date_time_end cannot be equal to date_time_start
             {'title': TITLE, 'date_time_start': DATE_TIME_START, 'date_time_end': DATE_TIME_START, 'color': COLOR},
-        ])
+        ], ids=["title is none", "title is blank", "star time is none", "end time is none",
+                "end time is not bigger then start time", "start time and end time are equal"]
+     )
```

Comment:
I would do each id in a separate line. 
in this way it is more readable and easy to understand which is belongs to which parametrize like:
```suggestion
        ], ids=[
            "title is none", 
            "title is blank", 
            "star time is none", 
            "end time is none",
            "end time is not bigger then start time", 
             "start time and end time are equal"
          ]
     )
```

Reply:
done 😄